homeassistant/components/sensor/yweather.py

```diff
@@ -130,12 +139,22 @@ def entity_picture(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
+        if self._day and self._date:
+            return {
```

Comment:
I didn't know how that's why I left it as it is. Could you please give me a example of how to solve it as you wish?